crates/accelerate/src/remove_identity_equiv.rs

```diff
@@ -140,6 +152,11 @@ fn remove_identity_equiv(
     for node in remove_list {
         dag.remove_op_node(node);
     }
+
+    if global_phase_update != 0. {
+        dag.add_global_phase(py, &Param::Float(global_phase_update))
+            .unwrap();
```

Comment:
The addition can only fail if the Parameter is not a float, so we can give a reason why this shouldn't fail (and if it does fail, then we know what assumption we had that's wrong 🙂)
```suggestion
            .unwrap("The global phase is guaranteed to be a float");
```

Reply:
Done in [68e6a7f](https://github.com/Qiskit/qiskit/pull/13785/commits/68e6a7fca205e994ea7e1b0d95a8c082a9644235). And of course, you mean `expect` instead of `unwrap`.